### ISSUE TITLE
Add props from Lottie component to underlying div container 

### ISSUE THREAD

**Issue by jamiehaywood** [Gamote/lottie-react#11]
**Is your feature request related to a problem? Please describe.**
If I give the `<Lottie />` component props, I would like to be able to add standard div props and for them to be injected into the underlying `<div>` container.

_Example:_
```tsx
    <Lottie
      animationData={animation}
      aria-busy="true"
      aria-label={label}
      aria-live="polite"
      label={label}
      role="alert"
      style={{ height: '1.167rem', margin: 'auto' }}
    />
```

**Describe the solution you'd like**
You could extend `LottieOptions` type to include`React.HTMLProps<HTMLDivElement>` and then inside the hook on L39: 

https://github.com/Gamote/lottie-react/blob/d96147bed92466b3c5d6e43338059f896ca0ca15/src/hooks/useLottie.tsx#L39 

you could then add a spread `{...rest}` and add them onto the View on line 255:
https://github.com/Gamote/lottie-react/blob/d96147bed92466b3c5d6e43338059f896ca0ca15/src/hooks/useLottie.tsx#L255
```
  const View = <div style={style} ref={animationContainer} {...rest} />;
```

**Describe alternatives you've considered**

**Additional context**

https://github.com/Gamote/lottie-react/pull/12

**Comment by jamiehaywood** [comment]
@Gamote sorry to pester - any chance of merging & publishing this? I'm waiting on this fix for work 😕

**Comment by Gamote** [comment]
Hi @jamiehaywood,

Yes I will merge it and publish it until the end of the day.

**Comment by Gamote** [comment]
@jamiehaywood This changes were published on NPM under the [v2.1.0](https://github.com/Gamote/lottie-react/releases/tag/v2.1.0) release.

**Comment by jamiehaywood** [comment]
Many thanks @Gamote.
